recipes/ncagg/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "0.8.12" %}
+
+package:
+  name: ncagg 
+  version: {{ version }}
+
+source:
+  url: https://github.com/5tefan/ncagg/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 49517f3bc6d3e6c2e476dda5e38a1646e331ea12a07eef049084faae0dc6f651
+
+build:
+  noarch: python
+  entry_points:
+    - "ncagg = ncagg.cli:cli"
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

You can run the tests as follows during the build.
```suggestion
  script: 
    - "{{ PYTHON }} -m pip install . -vv"
    - python -m unittest discover -s $SRC_DIR
```

---

Ah, interesting. Thanks!